app/models/pet.rb

```diff
@@ -2,5 +2,7 @@ class Pet < ApplicationRecord
 
   validates_presence_of :name, :age, :sex
   belongs_to :shelter
-
+  has_many :pet_applications, dependent: :destroy
+  has_many :applications, through: :pet_applications
+  
```

Comment:
I'm thinking after we finish the project, we can go over it and do one large check for things like extra spaces, and such.

Reply:
sounds good!